docker-compose/technical/docker-compose.technical.yml

```
@@ -110,3 +110,31 @@ services:
       - PGADMIN_DEFAULT_PASSWORD=admin
     volumes:
       - $PWD/../technical/servers_pgadmin.json:/pgadmin4/servers.json
+
+  prometheus:
+    profiles:
+      - all
+      - metrics
+    image: prom/prometheus
+    command:
+      - --enable-feature=exemplar-storage
+      - --config.file=/etc/prometheus/prometheus.yml
+    volumes:
+      - $PWD/../technical/prometheus.yml:/etc/prometheus/prometheus.yml:ro
+    ports:
+      - "8000:9090"
+
+  grafana:
+    profiles:
+      - all
+      - metrics
+    image: grafana/grafana
+    volumes:
+      - $PWD/../technical/grafana/config/datasources:/etc/grafana/provisioning/datasources:ro
+      - $PWD/../technical/grafana/config/dashboards:/etc/grafana/provisioning/dashboards:ro
+    environment:
+      - GF_AUTH_ANONYMOUS_ENABLED=true
+      - GF_AUTH_ANONYMOUS_ORG_ROLE=Admin
+      - GF_AUTH_DISABLE_LOGIN_FORM=true
+    ports:
+      - "7000:3000"
```

miss final return line
```suggestion
      - "7000:3000"

```

---

Done